build/ferretdb/all-in-one.Dockerfile

```diff
@@ -135,6 +135,8 @@ ENV POSTGRES_DB=ferretdb
 STOPSIGNAL SIGHUP
 ENTRYPOINT [ "/entrypoint.sh" ]
 
+HEALTHCHECK CMD /ferretdb ping
+
 # all-in-one hacks stop there
```

Comment:
Let's move `HEALTHCHECK` after that comment. Let's also add all flags (`--interval`, `--timeout`, etc) to make it easier to tweak later